widget/embedded/src/components/BlockchainList/BlockchainList.styles.ts

```diff
@@ -4,6 +4,7 @@ export const Container = styled('div', {
   display: 'flex',
   flexDirection: 'column',
   overflow: 'hidden',
+  textAlign: 'left',
```

Comment:
We should find a better way to handle alignment. can speak offline.

Reply:
@Ikari-Shinji-re 
Please check your changes with something this to make sure every thing works. I think we need more changes in parent container rather than just adding a text-align for the  parent.

```css
*,
div *,
span *,
input * {
    text-align: right;
}
```